Week2/queries.js

```diff
@@ -0,0 +1,68 @@
+const question1 = `
+SELECT
+   country.name AS country,
+   city.name AS capital 
+FROM
+   city,
+   country 
+WHERE
+   country.capital = city.id 
+   AND country.name = ? `;
+
+const question2 = `
+   SELECT
+      countrylanguage.language AS language 
+   FROM
+      country,
+      countrylanguage 
+   WHERE
+      countrylanguage.countrycode = country.code 
+      AND country.region = ? 
+   GROUP BY

```

Comment:
No need of group by here

Reply:
There are some duplicated regions so I group them.

Reply:
Ok got it, you could have used a distinct also on countrylanguage .